### DISCUSSION TITLE
pi400

### DISCUSSION THREAD

**Discussion by BaskaranProdigy** [Pi4J/pi4j#253] [General]
Iam building Project using Pi400 ,Input Power using External Power Suppy unit 
What is the Min Input and Max Input Voltage Range ?
Thanks
K.Baskaran

**Comment by FDelporte** [comment]
From the PiHut website:

The [Raspberry Pi 400](https://thepihut.com/products/raspberry-pi-400) uses the same power supply as a Raspberry Pi 4 - a 5.1V 3A USB-C power supply.

Remember - anything less than 5.1V 3A may cause performance and stability issues. You should not attempt to power your Raspberry Pi 400 from a computer USB port - always use dedicated power supplies built specifically for the Raspberry Pi.

**Reply by FDelporte** [reply]
https://support.thepihut.com/hc/en-us/articles/360015199058-How-do-I-power-a-Raspberry-Pi-400-

**Reply by BaskaranProdigy** [reply]
Hi
Thanks for your reply
For my Project ,iam using External PSU 
From PSU -Iam getting 5.5V ,Is the PI400 Can accept ?,
What is the Max Input Voltage for Pi400 ?
